sql/core/src/main/java/org/apache/spark/sql/execution/streaming/StateMessage.proto

```diff
@@ -0,0 +1,86 @@
+syntax = "proto3";
+
+package org.apache.spark.sql.execution.streaming.state;
```

Comment:
I think the Poc has this file twice too, if I recall correctly I needed a package prefix for Java (and not Python). Do you know if we can just use a single file, or do we need to duplicate the definitions for Java/Python?

Reply:
Yeah we can keep a single file, removed the one under python